components/dom_crawler.rst

```diff
@@ -327,6 +329,23 @@ methods to get more information about the selected link itself::
     page suffixed with ``#foo``. The return from ``getUri()`` is always a full
     URI that you can act on.
 
+Images
+~~~~~~
+
+To find an image by its ``alt`` attribute, use the ``selectImage`` method on an
+existing crawler. This returns a Crawler instance with just the selected
```

Comment:
We usually enclose class names with literas. Why do you not want to do this here?